pipeline/unified/channel/sign-release-package.yaml

```diff
@@ -34,8 +35,12 @@ jobs:
                 signConfigType: inlineSignParams
                 inlineOperation: ${{ parameters.inlineSignParams }}
 
-          - script: yarn update:electron-checksum signing-in-progress/${{ parameters.signedArtifactName }}
-            displayName: update electron checksum after signing
+          - script: node ./pipeline/scripts/remove-pkg.js signing-in-progress/${{ parameters.signedArtifactName }}
+            displayName: remove .pkg binary from zip file
+            condition: eq(variables['Agent.OS'], 'Darwin')
+
+          - script: yarn update:latest-yml signing-in-progress/${{ parameters.signedArtifactName }} ${{ parameters.platform }}
```

Comment:
Should we get rid of the `update:latest-yml` package.json entry and just call `node ./pipeline/scripts/etc.js` directly like on line 38? The more direct path feels a little nicer to me if this is the only place it's used; less unnecessary stuff in package.json and one less layer to have to follow to understand the pipeline